static/app/views/insights/browser/resources/components/charts/resourceLandingPageCharts.tsx

```diff
@@ -0,0 +1,241 @@
+import styled from '@emotion/styled';
+
+import {getInterval} from 'sentry/components/charts/utils';
+import {space} from 'sentry/styles/space';
+import type {PageFilters} from 'sentry/types/core';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {formatRate} from 'sentry/utils/formatters';
+import {EMPTY_OPTION_VALUE} from 'sentry/utils/tokenizeSearch';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {AVG_COLOR, THROUGHPUT_COLOR} from 'sentry/views/insights/colors';
+import Chart, {
+  ChartType,
+  useSynchronizeCharts,
+} from 'sentry/views/insights/common/components/chart';
+import ChartPanel from 'sentry/views/insights/common/components/chartPanel';
+import {useSpansQuery} from 'sentry/views/insights/common/queries/useSpansQuery';
+import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/insights/common/utils/constants';
+import {
+  getDurationChartTitle,
+  getThroughputChartTitle,
+} from 'sentry/views/insights/common/views/spans/types';
+import type {ModuleFilters} from 'sentry/views/insights/common/views/spans/useModuleFilters';
+import {ModuleName, SpanMetricsField} from 'sentry/views/insights/types';
+
+import {RESOURCE_THROUGHPUT_UNIT} from '../../settings';
+
+const {SPAN_SELF_TIME, SPAN_DESCRIPTION, SPAN_DOMAIN} = SpanMetricsField;
+
+const CHART_HEIGHT = 140;
+
+type Props = {
+  appliedFilters: ModuleFilters;
+  extraQuery?: string[];
+};
+
+type ChartProps = {
+  filters: ModuleFilters;
+  title: string;
+  extraQuery?: string[];
+};
+
+export function ResourceLandingPageCharts({appliedFilters, extraQuery}: Props) {
+  const moduleName = ModuleName.RESOURCE;
+  const {selection} = usePageFilters();
+
+  const eventView = getEventView(selection, appliedFilters);
+
+  if (extraQuery) {
+    eventView.query += ` ${extraQuery.join(' ')}`;
+  }
+
+  const {isPending} = useSpansQuery({
+    eventView,
+    initialData: [],
+    referrer: 'api.starfish.span-time-charts',
```

Comment:
```suggestion
    referrer: 'api.performance.resources.span-time-charts',
```

Additionally, I think ideally this comes from a `referrers.ts` file as the other modules do.

Reply:
I can update that in a follow-up! 👍🏻